server/controllers/assess/assessmentsController.ts

```diff
@@ -15,12 +15,18 @@ export default class AssessmentsController {
 
   index(): RequestHandler {
     return async (req: Request, res: Response) => {
-      const assessments = await this.assessmentService.getAll(req.user.token)
+      const { activeTab } = req.query
+      const statuses =
+        activeTab === 'awaiting_assessment' || !activeTab
+          ? (['in_progress', 'not_started', 'in_review'] as Array<AssessmentStatus>)
```

Comment:
I think it would be worth defining this array as a variable, typing it and then using it here and in the tests. Then we will be alerted if the `AssessmentStatus` type changes and we only have to update it one place

Reply:
Good plan!